IaC/AKS/README.md

```diff
@@ -512,18 +500,18 @@ az monitor log-analytics workspace create -g $Ngsa_Smoker_RG -l $Ngsa_Location -
 cd $REPO_ROOT/IaC/scripts
 
 # Run the shell script to install smoker instances with ACI
-./smokers.sh -n $Ngsa_Name -r $Ngsa_Smoker_RG -s $Ngsa_App_Endpoint
+./smokers.sh -n $Ngsa_Name -r $Ngsa_Smoker_RG -s $Ngsa_Https_App_Endpoint
```

Comment:
we're deploying webv to the clusters, not ACI (so that we can measure cluster to cluster latency)

Reply:
Gotcha, I'll scrap this out